solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -151,27 +152,42 @@ contract RandomBeacon is IRandomBeacon, Ownable {
     ///         operator affected.
     uint256 public dkgMaliciousResultNotificationRewardMultiplier;
 
+    /// @notice Calculated gas cost for submitting a DKG result. This will
+    ///         be refunded as part of the DKG approval process. It is in the
+    ///         submitter's interest to not skip his priority turn on the approval,
+    ///         otherwise the refund of the DKG submission will be refunded to
+    ///         other member that will call the DKG approve function.
```

Comment:
Let's be a bit more specific about what member we are talking about. If that's also the case for `WalletRegistry.sol`, let's update both in this PR:
```suggestion
    ///         another group member that will call the DKG approve function.
```

Reply:
[dd9ad55](https://github.com/keep-network/keep-core/pull/2860/commits/dd9ad55739999e3d5bdc88bdc68b84b1c20367f9)